assets/js/base/hooks/use-store-cart-items.js

```diff
@@ -0,0 +1,43 @@
+/** @typedef { import('@woocommerce/type-defs/hooks').StoreCartItems } StoreCartItems */
+
+/**
+ * External dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { CART_STORE_KEY as storeKey } from '@woocommerce/block-data';
+
+/**
+ * Internal dependencies
+ */
+import { useStoreCart } from './use-store-cart';
+
+/**
+ * This is a custom hook for loading the Store API /cart/ endpoint and
+ * actions for removing or changing item quantity.
+ * See also: https://github.com/woocommerce/woocommerce-gutenberg-products-block/tree/master/src/RestApi/StoreApi
+ *
+ * @return {StoreCartItems} An object exposing data and actions relating to cart items.
+ */
+export const useStoreCartItems = () => {
```

Comment:
I though in another pull we completely remove this hook in favor of `useStoreCartItem`.  Did it get added back in this pull for some reason? I did a search and `useStoreCartItems` is not being implemented anywhere, so I'd prefer to see this removed.

Reply:
Looks like this was a rebase error, totally didn't intend to bring it back!